lib/spack/spack/test/cvs_fetch.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+
+import pytest
+
+from llnl.util.filesystem import touch, working_dir, mkdirp
+
+import spack.repo
+import spack.config
+from spack.spec import Spec
+from spack.stage import Stage
+from spack.version import ver
+from spack.fetch_strategy import CvsFetchStrategy
+from spack.util.executable import which
+
+
+pytestmark = pytest.mark.skipif(
+    not which('cvs'),
```

Comment:
You should also check to make sure `expect` is installed if we need that for the tests

Reply:
The tests use a local repository, and thus don't need `expect`.